opportunity_job_app/src/api/authService.ts

```diff
@@ -9,7 +9,8 @@ export const AuthService = {
         const userDataString = JSON.stringify({
             token: response.data.token,
             id: response.data.id,
-            accountType: response.data.account_type
+            accountType: response.data.account_type,
+            account_id: response.data.account_id
```

Comment:
Ovde bih promenio key `accountType` u `account_type` da odgovara modelu na BE, tj. response-u koji dobijamo sa BE. Isto vazi i za `User` model kojeg smo definisali u `api/models/User.tsx`.